tests/functional_tests/test_get_discoveries_sqlite.py

```diff
@@ -0,0 +1,132 @@
+import os
+import tempfile
+import unittest
+
+import pandas as pd
+from credentialdigger.cli import cli
+from credentialdigger.client_sqlite import SqliteClient
+from parameterized import param, parameterized
+
+
+class TestGetDiscoveries(unittest.TestCase):
+
+    @classmethod
+    def setUpClass(cls):
+        # Set up sqlite database and CSV export temporary paths
```

Comment:
same comment as in the other file

Reply:
I have introduce a new seperate docstring.